tests/e2e/helper.py

```
@@ -18,6 +18,20 @@ def _delete_resource(service, resource):
         print(f"TEARDOWN - Unable to delete resource {resource}: {error.title}")  # noqa:  WPS421
 
 
+async def _finalize_async_resource(finalize, resource):
+    try:
+        await finalize(resource.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to finalize resource {resource}: {error.title}")  # noqa:  WPS421
```

Ruff (0.15.15)

[warning] Invalid rule code in `# noqa`: WPS421

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)